travis/install.sh

```diff
@@ -5,7 +5,7 @@
 set -ex
 
 source "${HOME}/INSTALL/bin/activate"
-pip install --upgrade pip wheel six setuptools nodeenv
```

Comment:
Since MarkupSafe is dependent on Jinja2, why don't you specify the version in the following areas rather than mentioning the version in Travis YML?
Travis thinks it looks clearer to focus only on testing and deployment, and collecting requirements related to the version.
I experimented with this in my fork Repository. (https://github.com/Digitalisx/grr/pull/1)
Feel free to leave your comments!

Reply:
Oh, I failed the final build, I think your method is right.
I'm sorry that I couldn't make a suggestion after checking a little more!
I hope your work goes well.

Reply:
Hi @Digitalisx, good point! Do you mind creating a PR with your proposal for me to review? :)